package.json

```diff
@@ -2,7 +2,7 @@
   "name": "html-select2",
   "description": "Match & splice a tokenized HTML stream with css selectors. Like html-select but maintained.",
   "homepage": "https://documentup.com/Kikobeats/html-select2",
-  "version": "1.0.1",
```

Comment:
Can you remove this line? I have a release process for incrementing the version automagically 🙂